src/components/StatueDetailContainer.js

```diff
@@ -1,14 +1,21 @@
 import React, { Component } from "react";
-import style from "./StatueDetailContainer.module.css";
+import "./StatueDetailContainer.module.css";
 
 export class StatueDetailContainer extends Component {
   render() {
     const { name, details } = this.props.statue;
 
     return (
-      <div className={style.container}>
-        <div>{name}</div>
-        <div>{details}</div>
+      <div className="col-lg-8 col-md-6 col-sm-12">
+        <div className="row mt-2">
+          <h2 className="mb-4">{name}</h2>
+          <div className="col-sm-5">
```

Comment:
try this: `col-sm-12 col-md-5`

Reply:
This is still as it was before, and is causing a little bit of trouble with the image at the small breakpoint, the text underneath is 12 columns across, but the image is tiny and should also be 12 columns, becoming 5 columns at the medium breakpoint to fit i nthe with text which is 7 at medium :)

Reply:
This is still the same?